src/couch_expiring_cache/src/couch_expiring_cache_fdb.erl

```diff
@@ -0,0 +1,107 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_expiring_cache_fdb).
+
+-export([
+    insert/5,
+    lookup/2,
+    clear_expired_range/2
+]).
+
+
+-define(XC, 53). % coordinate with fabric2.hrl
```

Comment:
I'm a bit nervous that we have it here instead of fabric2.hrl, it could be easy for someone else to later use the namespace for something else. Also `XC` isn't a particluarly clean name. I have no idea what it means. Could you expand it a little.

Reply:
I agree that it's not a great solution. I just copied this from [couch_jobs](https://github.com/apache/couchdb/blob/prototype/fdb-layer/src/couch_jobs/src/couch_jobs.hrl#L29), but maybe we should just make it a practice to directly modify `fabric2.hrl` as necessary?

Also, expanded `XC` to `EXPIRING_CACHE` in https://github.com/apache/couchdb/pull/2331/commits/7144cebc31593e53625b94563965c63a8c4039fa